browser/ui/views/frame/vertical_tab_strip_widget_delegate_view.cc

```diff
@@ -137,6 +139,10 @@ VerticalTabStripWidgetDelegateView::VerticalTabStripWidgetDelegateView(
   host_view_observation_.Observe(host_);
   widget_observation_.AddObservation(host_->GetWidget());
 
+  // At this point, Browser hasn't finished its initialization. In order to
+  // access some of its member, we should observe BrowserList.
```

Comment:
Q) Don't we have to care the condition that browser is already added to the browser list here?

Reply:
Yeah, makes sense. Let's add some DCHECKs